StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentLogoView.swift

```diff
@@ -47,6 +53,31 @@ final class ConsentLogoView: UIView {
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    func animateDots() {
```

Comment:
is there any concern if this ends up being called multiple times without the previous animation having been removed?

Reply:
good question! I added some extra-duplicate-calls to `animateDots` just to see what happens and looks to work OK. That being said, I'll add an extra `multipleDotView.layer.removeAllAnimations()` just in case.